genpepindex.py

```diff
@@ -36,7 +36,7 @@ def main(argv):
     peps = []
     if os.path.isdir(path):
         for file_path in os.listdir(path):
-            if file_path == 'pep-0000.txt':
+            if file_path == 'pep-0000.rst':
```

Comment:
Best to switch this to `file_path.startswith('pep-0000.')` so old text files also get ignored.